app/components/DropDownProducts.jsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import Popover from 'material-ui/Popover';
+import Menu from 'material-ui/Menu';
+import IconMenu from 'material-ui/IconMenu';
+import MenuItem from 'material-ui/MenuItem';
+import IconButton from 'material-ui/IconButton';
+import Divider from 'material-ui/Divider';
+import ArrowDropRight from 'material-ui/svg-icons/navigation-arrow-drop-right';
+import MenuIcon from 'material-ui/svg-icons/navigation/menu';
+import Search from 'material-ui/svg-icons/action/search';
+import TextField from 'material-ui/TextField';
+import {RadioButton, RadioButtonGroup} from 'material-ui/RadioButton';
+import {browserHistory, Link} from 'react-router';
+
+export default class DropDownProducts extends React.Component {
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      open: false,
+      radio: 'name'
+    };
+  };
+
+
+
+  handleTouchTap = (event) => {
```

Comment:
As far as I understand this is a syntax error...as in: you can't have an `=` assignment in a class declaration. I see that the build is working but I am not sure why its working. All the handler functions should be :

```.js
handleTouchTap(event) {
  // ...
}
```